### DISCUSSION TITLE
Alerting us of Errors past a certain threshold

### DISCUSSION THREAD

**Discussion by larsonel** [nominal-systems/dmi#27] [Ideas]
Is there a way to ask the DMI Admin UI to send an alert or e-mail a group of us if our integrations start receving 5xx and 4xx above a certain threshold? 

I.e. - If Wisdom has >300 errors "today" then let us know. 

Since we don't have a support team constantly monitoring the integration, some notification that we should check out a potential problem would be helpful

**Comment by monsieurBelbo** [comment]
This is not currently possible, but an "Alerts" feature would be interesting. We should be able to support creating "Alert Groups" (a list of email contacts) and "Alerts" which is basically when a certain metric crosses a certain threshold on a given time period, e.g. "Wisdom Panel external requests in error greater than 300 per day". When an Alert crosses its threshold, then an email is sent to the Alert Group with a link to the Alert details.
